README.md

```diff
@@ -112,6 +112,30 @@ _must_ detect whether they're present or not before using them. See [this issue]
 and [this wiki page](https://github.com/silverstripe-australia/silverstripe-queuedjobs/wiki/Defining-queued-jobs) for 
 more information
 
+## Memory limit configuration
+
+By default this task will run until it comes without 12MB of the memory limit configured by PHP, giving the application
```

Comment:
If php has a memory limit of 132MB, and we have left the buffer at 12mb, it will continue executing until it uses 120mb (132 - 12).

It's confusing because I should have said "within" not "without"... I'll fix.